lib/conf.ml

```diff
@@ -15,5 +15,5 @@
  *
  *)
 
-let user_agent = "ocaml-cohttp/0.20.2"
-let version = "0.20.2"
+let user_agent = "ocaml-cohttp/0.21.0"
+let version = "0.21.0"
```

Comment:
Agreed. I will fix this.

On 05/23, David Sheets wrote:
> > @@ -15,5 +15,5 @@
> >   *
> >   *)
> >  
> > -let user_agent = "ocaml-cohttp/0.20.2"
> > -let version = "0.20.2"
> > +let user_agent = "ocaml-cohttp/0.21.0"
> > +let version = "0.21.0"
> 
> Seems odd that `user_agent` doesn't use `version` here...
> 
> ---
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/mirage/ocaml-cohttp/pull/484/files/3c072fcb0bd2a62ffe91f9f829d76e6d7989e3c3#r64190081